### DISCUSSION TITLE
Displaying a /dev/fb#/ Framebuffer?

### DISCUSSION THREAD

**Discussion by RDGInfoSys** [i3/i3#5179] [Q&A]
Is it possible to grab what's being written to a /dev/fb#/ framebuffer and display it in a pane?

Specifically, I have a process running on a RPi4 that's been written to provide a heads-up status display on an LCD connected to the SPI "port". I'd like to grab/redirect what's being written to the framebuffer and display it in a window/container on the main display.

**Comment by neg-serg** [comment]
It's definitely unrelated to i3.

First of all you need to try cat /dev/fb# > fb.bin and check that file is not empty.
Then use ffmpeg to convert it from .bin to something reasonable. It's not fast operations as I understand.
